docs/web/supported-browsers-and-frameworks.mdx

```diff
@@ -21,6 +21,90 @@ However, there are some gotchas to be aware of as well as common errors that you
 your stack.  The issues vary depending on the tooling in your project.  This page explains some helpful workarounds
 with links to examples.
 
+## Fetch API `keepalive`
```

Comment:
I don't think this page is well suited to surface this information. 

There are also many other fetch options that can be set this way. All the ones listed [here](https://developer.mozilla.org/en-US/docs/Web/API/fetch#options), except for "body", "headers", and "signal", which we need to control (but expose through other means).

The [Interceptors](https://connectrpc.com/docs/web/interceptors/) page could be expanded to explain it. The [Choosing a protocol](https://connectrpc.com/docs/web/choosing-a-protocol/) page could be expanded to show the option for a custom `fetch` function.

It might be worth adding a dedicated page "Fetch API", where we can explain both features with a bit more details, and link to from the other pages (including the [SSR](https://connectrpc.com/docs/web/ssr/#server-load-functions) page, where we give an example to BYO fetch).